packages/api-core/src/plugins/fetchers.ts

```diff
@@ -0,0 +1,9 @@
+import { wpcom } from '../wpcom-fetcher';
+import type { PluginsResponse } from './types';
+
+export async function fetchUserSitesPlugins(): Promise< PluginsResponse > {
```

Comment:
We're naming the directory based on the real API route (paths). Could we rename this to `src/me-sites-plugins`?

Reply:
Renamed.

Reply:
or for example something like `domain-forwarding` which has a fetcher:

```
export function fetchDomainForwarding( domainName: string ): Promise< DomainForwarding[] > {
	return wpcom.req.get( `/sites/all/domain/${ domainName }/redirects`, { 'new-endpoint': 'true' } );
}
```

Would we rename that to `sites-all-domain-redirects`? and would we also update the function name from `fetchDomainForwarding` to something else?